collectors/monitors.go

```diff
@@ -371,6 +503,30 @@ func (m *MonitorCollector) collect() error {
 
 	m.NodesinQuorum.Set(float64(len(stats.Quorum)))
 
+	// Ceph versions, one loop for each daemon.
+	// In a consistent cluster, there will only be one iteration (and label set) per daemon.
+	for daemon, vers := range versions {
+		for version, num := range vers {
+			// We have a version, which is something like the following, how we want to map it
+			// ceph version 12.2.13-30-aabbccdd (c5b1fd521188ddcdedcf6f98ae0e6a02286042f2) luminous (stable)
+			//              version_tag          sha1                                      release_tag
+			res := versionRegexp.FindStringSubmatch(version)
+			if len(res) != 4 {
+				m.CephVersions.WithLabelValues(daemon, "unknown", "unknown", "unknown").Set(num)
```

Comment:
If we have multiple of these within the daemon, they'll stomp on each other. Not sure if it really matters enough to account for that, though.

Reply:
Yeah, if it's unknown, it might not be a release that we should be trying to support (older than Luminous, or newer than anything what's available with today).